DeleteUserDataActivity/utils.ts

```diff
@@ -11,6 +14,21 @@ import {
   QueryFailure
 } from "./types";
 
+// Cosmos Errors
+export enum CosmosErrorsTypes {
+  "OSMOS_EMPTY_RESPONSE" = "OSMOS_EMPTY_RESPONSE",
+  "OSMOS_CONFLICT_RESPONSE" = "OSMOS_CONFLICT_RESPONSE",
+  "OSMOS_DECODING_ERROR" = "OSMOS_DECODING_ERROR",
+  "OSMOS_ERROR_RESPONSE" = "OSMOS_ERROR_RESPONSE"
```

Comment:
typo?!
```suggestion
  "COSMOS_EMPTY_RESPONSE" = "COSMOS_EMPTY_RESPONSE",
  "COSMOS_CONFLICT_RESPONSE" = "COSMOS_CONFLICT_RESPONSE",
  "COSMOS_DECODING_ERROR" = "COSMOS_DECODING_ERROR",
  "COSMOS_ERROR_RESPONSE" = "COSMOS_ERROR_RESPONSE"
```

Reply:
Definitely 😓  Thanks!!

Reply:
I've incorporated a test to verify the accuracy of enum elements by defining a tuple object containing all possible literal values of the `kind` property within `CosmosError`. By doing so, the test will signal failure upon any addition of a new 'kind' to `CosmosError`, prompting us to update the enum accordingly.
See [[#IOPID-1450] add tests for isCosmosErrors](https://github.com/pagopa/io-functions-admin/pull/231/commits/2e0f07e641fb16e12ad322da065762119eea0fb1)